src/main/java/core/basesyntax/model/Fruit.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.model;
+
+public enum Fruit {
```

Comment:
make a class Fruit instead of enum

Reply:
Ye, I'm tried to do this. Class Fruit with enum inside. Wrong way. OK, let's try again

Reply:
Wrong way. Fruit class and Fruit enum deleted